Source/Core/DolphinWX/Main.cpp

```diff
@@ -549,14 +548,6 @@ void Host_ConnectWiimote(int wm_idx, bool connect)
 void Host_ShowVideoConfig(void* parent, const std::string& backend_name,
                           const std::string& config_name)
 {
-	if (backend_name == "Direct3D" || backend_name == "OpenGL")
-	{
-		VideoConfigDiag diag((wxWindow*)parent, backend_name, config_name);
-		diag.ShowModal();
-	}
-	else if (backend_name == "Software Renderer")
-	{
-		SoftwareVideoConfigDialog diag((wxWindow*)parent, backend_name, config_name);
-		diag.ShowModal();
-	}
+	VideoConfigDiag diag((wxWindow*)parent, backend_name, config_name);
+	diag.ShowModal();
```

Comment:
RFC: There are lots of options available without any effect. Is it fine as it is, or shall they be hidden in any way?